sdk/python/feast/entity.py

```diff
@@ -30,8 +30,25 @@
 class Entity:
     """
     Represents a collection of entities and associated metadata.
+
+    Args:
+        name: Name of the entity.
+        value_type (optional): The type of the entity, such as string or float.
+        description (optional): Additional information to describe the entity.
+        join_key (optional): A property that uniquely identifies different entities
+            within the collection. Used as a key for joining entities with their
+            associated features. If not specified, defaults to the name of the entity.
+        labels (optional): User-defined metadata in dictionary form.
     """
 
+    _name: str
+    _value_type: ValueType
+    _description: str
+    _join_key: str
+    _labels: MutableMapping[str, str]
```

Comment:
nit, `MutableMapping` is a proto concept. we should be using `dicts` in the python classes and only deal with MutableMapping (if at all) in the `to_proto` or `from_proto` methods.

I don't think we *need* to address that in this PR, but i think it should be fixed at some point.

Reply:
Okay, I'll try to switch this to `dict`s.